integrations/client/test_delphi_epidata.py

```diff
@@ -17,6 +17,14 @@
 from delphi_utils import Nans
 
 
+# TODO replace these real geo_values with fake values, and use patch and mock to mock the return values of 
+# delphi_utils.geomap.GeoMapper().get_geo_values(geo_type) in parse_geo_sets() of _params.py
+
+global fips, msa # add two global lists of valid geo_values of these types to pass the validation in parse_geo_sets() of _params.py
+
```

Comment:
using the `global` keyword isnt necessary here, and many people try to avoid it completely if possible
```suggestion

```